invenio_accounts/ext.py

```diff
@@ -84,9 +97,27 @@ def init_app(self, app, use_celery=True, sessionstore=None,
                  register_blueprint=True):
         """Flask application initialization.
 
+        The following actions are executed:
+
+        #. Initialized the configuration.
```

Comment:
I would put all the following verbs either in the infinite form (Initialize, Monkey-patch, Create, confirm) like "register" line 112, or in the third-person present form (Initializes, Monkey-patches, Creates, registers) like "confirms" line 111.